transaction_essence.go

```diff
@@ -212,3 +213,14 @@ func (u *TransactionEssence) syntacticallyValidate(protoParams *ProtocolParamete
 
 	return nil
 }
+
+func (u *TransactionEssence) WorkScore(workScoreStructure *WorkScoreStructure) WorkScore {
+	// NetworkID, CreationTime
```

Comment:
`workScoreStructure.FactorData.Multiply(util.NumByteLen(TransactionEssenceNormal))` is missing here